packages/react-components/react-datepicker-compat/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -66,11 +66,6 @@ export type DatePickerProps = ComponentProps<Partial<DatePickerSlots>> & {
    */
   disabled?: boolean;
 
-  /**
-   * Aria Label for TextField of the DatePicker for screen reader users.
-   */
-  ariaLabel?: string;
-
```

Comment:
This was removed to conform with conformance tests since in v9 we use kebab cased aria attributes. This is enforced by isConformant.